openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -242,13 +242,10 @@ Note: In some credential types, it is not desirable to include an expiration dat
 
 ## Issuer identification and key resolution to validate an issued Credential {#issuer-key-resolution}
 
-This profile supports two ways to represent and resolve the key required to validate the issuer signature of an SD-JWT VC, the web PKI-based key resolution and the x.509 certificates.
+This profile supports one mandatory way to represent and resolve the key required to validate the issuer signature of an SD-JWT VC, x.509 certificates. This MUST be supported by all entities (Issuer, Wallet, Verifier):
```

Comment:
Would this be a better wording?
```suggestion
This profile supports one way that is mandatory to support to represent and resolve the key required to validate the issuer signature of an SD-JWT VC, x.509 certificates. This MUST be supported by all entities (Issuer, Wallet, Verifier):
```

Reply:
Maybe this?

```suggestion
This profile mandates the support for x.509 certifications to represent and resolve the key required to validate the issuer signature of an SD-JWT VC. This MUST be supported by all entities (Issuer, Wallet, Verifier):
```

But I think we actually need to combine this paragraph and the single bullet into a single hopefully simpler paragraph.